projects/github-actions/repo-gardening/src/tasks/add-labels/index.js

```diff
@@ -336,20 +337,45 @@ async function addLabels( payload, octokit ) {
 	// If the PR title includes the word "revert", mark it as such.
 	const isRevert = title.toLowerCase().includes( 'revert' );
 
-	const labels = await getLabelsToAdd( octokit, owner.login, name, number, isDraft, isRevert );
+	const currentLabels = payload.pull_request.labels.map( l => l.name );
```

Comment:
What do you think about getting `labels` directly from the start by destructuring at the top of the function, with `const { draft, title } = pull_request;`. I think it makes things a bit more readable on the long term, and help if we later need to get labels for something else.

Reply:
The latest edition pulls from the current PR, so a good comment but no longer relevant. :^)